terragrunt/log_archive/legacy_archives/main.tf

```diff
@@ -0,0 +1,19 @@
+module "aws-landing-zone-logs_bucket" {
+  source = "github.com/cds-snc/terraform-modules?ref=v3.0.19//S3_log_bucket"
+
+  providers = {
+    aws = aws.log_archive
+  }
```

Comment:
There is no log_archive provider in this module, you can leave this out and it will pick up the default provider.